thinc/extra/wrappers.py

```diff
@@ -93,3 +94,48 @@ def use_params(self, params): # pragma: no cover
         yield
         self.from_bytes(backup)
 
+class PyTorchWrapperRNN(PyTorchWrapper):
+    '''Wrap a PyTorch RNN model
+    '''
+    def begin_update(self, x_data, h_0=None, drop=0., sgd=None):
+        '''Return the output of the wrapped PyTorch model for the given input,
+        along with a callback to handle the backward pass.
+        '''
+        x_var = torch.autograd.Variable(torch.Tensor(x_data),
+                                        requires_grad=True)
+        # Make prediction
+        out, h_n = self._model(x_var, h_0)
+        # Shapes will be:
+        # out = seq_len, batch, hidden_size * num_directions
+        # h_n = num_layers * num_directions, batch, hidden_size
+
+        def backward_pytorch(dy_data, sgd=None):
```

Comment:
I've learned from painful experience that these closures need specific names, or the tracebacks get pretty ugly. So we want this to be called `backward_pytorch_rnn`